crates/router/src/core/payment_methods/vault.rs

```diff
@@ -381,6 +382,47 @@ impl Vault {
 }
 
 //------------------------------------------------TokenizeService------------------------------------------------
+pub fn get_key_id(keys: &Jwekey) -> &str {
+    let key_identifier = "1"; // [#46]: Fetch this value from redis or external sources
+    if key_identifier == "1" {
+        &keys.locker_key_identifier1
+    } else {
+        &keys.locker_key_identifier2
+    }
+}
+
+#[cfg(feature = "basilisk")]
+pub async fn get_locker_jwe_keys(
+    keys: &Jwekey,
+) -> CustomResult<(String, String), errors::EncryptionError> {
```

Comment:
Should be have them as `Secret<(String, String)>` or `StrongSecret<(String, String)>`?

Reply:
not required